SECURITY.md

```diff
@@ -0,0 +1,33 @@
+# Security Policy
+
+## Supported Versions
+
+We are currently providing security updates to the following versions: 
+
+| Version | Supported          |
+| ------- | ------------------ |
+| 0.21.x   | :white_check_mark: |
+| 0.20.x   | :white_check_mark: |
+| 0.19.x   | :x: |
+| 0.18.x   | :x: |
+| < 0.18   | :x: |
+
+## Reporting a Vulnerability
+
+We will use [keybase](https://keybase.io) as the vehicle for reporting security issues as that gives us a
+much better feedback loop. We want to work with the reporter to make sure we understand the reported issue.
+
+If the vulnerability is found to be true we will release a patch version of our software
+and make sure that finds it way to Maven Central before we push the patch to github.
```

Comment:
Is it worth mentioning that source jars are still available for every binary on Maven Central?